x-pack/plugins/security_solution/public/management/pages/policy/view/tabs/blocklists_translations.ts

```diff
@@ -12,115 +12,116 @@ export const POLICY_ARTIFACT_BLOCKLISTS_LABELS = Object.freeze({
             values: { count: updatedExceptions.length },
           }
         )
       : i18n.translate(
           'xpack.securitySolution.endpoint.policy.blocklists.layout.flyout.toastSuccess.textSingle',
           {
-            defaultMessage: '"{name}" has been added to your blocklist list.',
+            defaultMessage: '"{name}" blocklist has been added to your list.',
             values: { name: updatedExceptions[0].name },
           }
         ),
   emptyUnassignedTitle: i18n.translate(
     'xpack.securitySolution.endpoint.policy.blocklists.empty.unassigned.title',
-    { defaultMessage: 'No assigned blocklists' }
+    { defaultMessage: 'No assigned blocklist entries' }
   ),
   emptyUnassignedMessage: (policyName: string): string =>
     i18n.translate('xpack.securitySolution.endpoint.policy.blocklists.empty.unassigned.content', {
       defaultMessage:
-        'There are currently no blocklists assigned to {policyName}. Assign blocklists now or add and manage them on the blocklists page.',
+        'There are currently no blocklist entries assigned to {policyName}. Assign blocklist entries now or add and manage them on the blocklist page.',
       values: { policyName },
     }),
   emptyUnassignedPrimaryActionButtonTitle: i18n.translate(
     'xpack.securitySolution.endpoint.policy.blocklists.empty.unassigned.primaryAction',
     {
-      defaultMessage: 'Assign blocklists',
+      defaultMessage: 'Assign blocklist entry',
     }
   ),
   emptyUnassignedSecondaryActionButtonTitle: i18n.translate(
     'xpack.securitySolution.endpoint.policy.blocklists.empty.unassigned.secondaryAction',
     {
-      defaultMessage: 'Manage blocklists',
+      defaultMessage: 'Manage blocklist entries',
     }
   ),
   emptyUnexistingTitle: i18n.translate(
     'xpack.securitySolution.endpoint.policy.blocklists.empty.unexisting.title',
-    { defaultMessage: 'No blocklists exist' }
+    { defaultMessage: 'No blocklist entries exist' }
   ),
   emptyUnexistingMessage: i18n.translate(
     'xpack.securitySolution.endpoint.policy.blocklists.empty.unexisting.content',
     {
-      defaultMessage: 'There are currently no blocklists applied to your endpoints.',
+      defaultMessage: 'There are currently no blocklist entries applied to your endpoints.',
     }
   ),
   emptyUnexistingPrimaryActionButtonTitle: i18n.translate(
     'xpack.securitySolution.endpoint.policy.blocklists.empty.unexisting.action',
-    { defaultMessage: 'Add blocklists' }
+    { defaultMessage: 'Add blocklist entry' }
   ),
   listTotalItemCountMessage: (totalItemsCount: number): string =>
     i18n.translate('xpack.securitySolution.endpoint.policy.blocklists.list.totalItemCount', {
-      defaultMessage: 'Showing {totalItemsCount, plural, one {# blocklist} other {# blocklists}}',
+      defaultMessage:
+        'Showing {totalItemsCount, plural, one {# blocklist} other {# blocklist entries}}',
```

Comment:
are we keeping with the blocklist + entry pattern here too for the singular case?

Reply:
Good catch, will fix it